ui/lib/apps/KeyViz/components/KeyViz.tsx

```diff
@@ -174,6 +182,14 @@ const KeyViz = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [])
 
+  const onChangeMetricType = useCallback(
+    (v: DataTag) => {
+      setMetricType(v)
+      resetAutoRefresh()
```

Comment:
I believe it is intentional that the there is no need to reset the refresh when metric type is changed, since there will be no data fetching operations during this.

Reply:
It looks better to recover the remaining seconds. And there's data fetching logic caused by metric type at L195-L200. Is this as expected? PTAL.